src/Kafka/Avro/SchemaRegistry.hs

```diff
@@ -78,6 +91,62 @@ sendSchema sr subj sc = do
   where
     schemaName = fullTypeName sc
 
+getVersions :: MonadIO m => SchemaRegistry -> Subject -> m (Either SchemaRegistryError [Word32])
+getVersions sr (Subject sbj) = do
+  let url = (srBaseUrl sr) ++ "/subjects/" ++ unpack sbj ++ "/versions"
+  resp <- liftIO $ Wreq.getWith wreqOpts url
+  pure $ bimap wrapError (view Wreq.responseBody) (Wreq.asJSON resp)
+  where
+    wrapError :: SomeException -> SchemaRegistryError
+    wrapError someErr = case fromException someErr of
+      Nothing      -> SchemaRegistrySendError (displayException someErr)
+      Just httpErr -> fromHttpError httpErr (\_ -> SchemaRegistrySendError (displayException someErr))
+
+checkCompatibility :: MonadIO m => SchemaRegistry -> Subject -> Word32 -> Schema -> m (Either SchemaRegistryError Bool)
+checkCompatibility sr (Subject sbj) version schema = do
+  let url  = (srBaseUrl sr) ++ "/compatibility/subjects/" ++ unpack sbj ++ "/versions/" ++ show version
+  resp     <- liftIO $ Wreq.postWith wreqOpts url (toJSON $ RegisteredSchema schema)
+  wrapped  <- pure $ bimap wrapError (view Wreq.responseBody) (Wreq.asValue resp)
+  either (return . Left) getCompatibility wrapped
+  where
+    getCompatibility :: MonadIO m => Value -> m (Either e Bool)
+    getCompatibility = liftIO . maybe (throwIO $ Wreq.JSONError "Missing key 'is_compatible' in Schema Registry response") (return . return) . viewCompatibility
+
+    viewCompatibility :: Value -> Maybe Bool
+    viewCompatibility (Object obj) = HM.lookup "is_compatible" obj >>= toBool
+    viewCompatibility _            = Nothing
+
+    toBool :: Value -> Maybe Bool
+    toBool (Bool b) = Just b
+    toBool _        = Nothing
+
+    wrapError :: SomeException -> SchemaRegistryError
```

Comment:
`wrapError` seems to be the same for several functions. Consider promote it to a module-level function and use in all the places